src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -175,6 +61,55 @@ const Conversation: React.FC<ConversationProps> = props => {
     />
   ))
 
+  // Pagination Scroll Logic
+  const [fetchingMore, setFetchingMore] = useState(false)
+  const [messagesTop, setMessagesTop] = useState(null)
+  const scrollContainer = useRef(null)
+
+  useEffect(() => {
+    if (!("IntersectionObserver" in window)) return
+    const observer = new IntersectionObserver(
+      entries => {
+        const first = entries[0]
+        if (first.isIntersecting && !initialMount.current) {
+          _loadMore()
+        }
+      },
+      { threshold: 0, rootMargin: `150px` }
+    )
+
+    const currentMessagesTop = messagesTop
+    const currentObserver = observer
+    if (currentMessagesTop) {
+      currentObserver.observe(currentMessagesTop)
+    }
+
+    // Cleanup
+    return () => {
+      if (currentMessagesTop) {
+        currentObserver.unobserve(currentMessagesTop)
+      }
+    }
+  }, [messagesTop])
+
+  const _loadMore = (): void => {
```

Comment:
**question:** What's the utility of the leading `_` here? Is it to avoid shadowing or to denote the function being private?

Reply:
Frankly, I saw it on the example on their docs, so I assumed it was convention. I'll take it out 🙃